src/XIVLauncher.Common.Unix/Compatibility/CompatibilityTools.cs

```diff
@@ -181,6 +192,13 @@ private Process RunInPrefix(ProcessStartInfo psi, string workingDirectory, IDict
             ldPreload = ldPreload.Equals("") ? "libgamemodeauto.so.0" : ldPreload + ":libgamemodeauto.so.0";
         }
 
+        if (isLaunchingFfxivExe && overlayFix.Contains("true"))
+        {
+            // If this is the actual RunInPrefix that is running FFXIV (not wine setup commands) then we should
+            // activate the steam overlay lib but otherwise we should NOT activate the SteamOverlay
+            ldPreload = ldPreload.Equals("") ? "/home/mbutler/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so" : ldPreload + ":/home/mbutler/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so";
```

Comment:
yeah, I agree. I'm not sure how to do that. maybe there is a Steam environment variable so we can make it something like `$STEAM_PATH/ubuntu12_64/gameoverlayrenderer.so`

Originally I thought when the file path said `Unix` that just meant Linux based systems but I guess this is also used on macOS?

Reply:
For Linux you could use the xdg spec. Use `$XDG_DATA_HOME` if it's set, otherwise use the default value of `$HOME/.local/share`. This probably wouldn't work with flatpak steam, though. So maybe something like this?
```suggestion
            string xdg_data_home = Environment.GetEnvironmentVariable("XDG_DATA_HOME") ?? Environment.GetEnvironmentVariable("HOME") + "/.local/share";
            string overlayPath = xdg_data_home + "/Steam/ubuntu12_64/gameoverlayrenderer.so";
            ldPreload = ldPreload.Equals("") ? overlayPath : ldPreload + ":" + overlayPath;
```

Reply:
Thank you for the suggestion. For now, I'm going to close this pull request because I no longer seem to experience the overlay problem and I'm using the `master` branch of the repository. Perhaps it's because I started using gamescope.